aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,343 @@
+    STAGING = "staging"
+    PRODUCTION = "production"
+
+
+class CloudFrontInvalidationError(Exception):
+    """CloudFront invalidation operation error"""
+
+
+class CloudFrontCacheInvalidator:
+    """Handles CloudFront cache invalidation operations"""
+
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)